components/admin_console/permission_schemes_settings/edit_post_time_limit_modal/edit_post_time_limit_modal.jsx

```diff
@@ -91,62 +91,67 @@ export default class EditPostTimeLimitModal extends React.Component {
                     </h4>
                 </Modal.Header>
                 <Modal.Body>
-                    <FormattedMessage
+                    <FormattedHTMLMessage
```

Comment:
Apparently there was a dev decision to stop using `FormattedHTMLMessage`

Reply:
@mkraft How can we format help text then? ie bold and links